docs/README.md

```diff
@@ -47,7 +47,7 @@ re-deployed.
 ### Preview Markdown `content` (*.md)
 
 The majority of this can be done with any markdown renderer (typically built into or
-a via plug-in for IDE's and editors). The rendered output will be very similar to what Hugo will
+via a plug-in for IDE's and editors). The rendered output will be very similar to what Hugo will
```

Comment:
[nit] Isn't it "IDEs" rather than "IDE's"? Just since your touching the line anyways 🤔